src/Emulator/Tabs/ConfigureTab/ConfigureTab.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+import "./ConfigureTab.css";
+import DeviceContainer from "./DeviceContainer/DeviceContainer";
+
+// Sample responses from currently non-existent API
```

Comment:
These are really useful! Both as documentation and for testing by [mocking API calls](https://www.leighhalliday.com/mock-fetch-jest)